### PR TITLE
Updates helio, selene maints

### PR DESCRIPTION
**Pull request by Horatio22**
## About The Pull Request

Makes some of helio's maints more interesting. Namely, the ramen room is now a weeb den with slightly fewer ramen cups.
image

Also moves the TEG from helio to the asteroid
image
 and changes the TEG room to this:
image

TEG is fully removed from selene.

## Why It's Good For The Game

TEG is only used for grief and doesn't really have a place as a roundstart item on Fulp maps. Putting it on the asteroid A- makes it much more difficult to obtain roundstart and B- gives people a reason to go to the asteroid on helio.

## Changelog
:cl:
balance: TEG is only on helio and is much less accessible at roundstart.
/:cl:

### CODE REVIEW

**Comment by FernandoJ8** [comment]
SO TRUE

**Comment by JohnFulpWillard** [comment]
I thought the TEG didn't even work, an engineer tried to run it and they told me it wasn't functional lol

**Comment by Horatio22** [comment]
> I thought the TEG didn't even work

Do I look like I know how engineering works

File: _maps/map_files/SeleneStation/SeleneStation.dmm [hunk 1]
```diff
@@ -17185,11 +17185,10 @@
 /turf/open/floor/iron,
 /area/station/hallway/primary/fore)
 "fKc" = (
-/obj/structure/closet/crate/grave,
-/obj/item/circuitboard/machine/circulator,
-/obj/item/circuitboard/machine/generator,
-/obj/item/circuitboard/machine/circulator,
-/turf/open/floor/grass,
```
Comment (Pepsilawn) [inline _maps/map_files/SeleneStation/SeleneStation.dmm:17192]: NO

Comment (Horatio22) [reply _maps/map_files/SeleneStation/SeleneStation.dmm:17192]: It's the grave subtype that kills you, if that helps

**Pull request closed without merging**